src/main/resources/assets/biomancy/lang/zh_cn.json

```diff
@@ -1,605 +1,641 @@
+  "tooltip.biomancy.action.enable_awakened_mode": "唤醒",
+  "tooltip.biomancy.action.disable_awakened_mode": "重新休眠",
+  "ability.biomancy.bleed_proc": "内脏切除",
+  "ability.biomancy.bleed_proc.desc": "触发时，对目标施加出血效果（最高II）",
+  "ability.biomancy.blood_explosion": "血爆",
+  "ability.biomancy.blood_explosion.desc": "攻击出血敌人时，额外造成最大生命10%的伤害",
+  "ability.biomancy.acid_coat": "酸浸",
+  "ability.biomancy.acid_coat.desc": "右击时使刀刃被酸液覆盖，两次攻击后失效。\n造成完整伤害后施加4s碎甲I、3s酸液腐蚀I。\n暴击时碎甲效果提高至II。",
+  "ability.biomancy.thorny_hide": "荆棘表皮",
+  "ability.biomancy.thorny_hide.desc": "成功防御时，对攻击者造成荆棘伤害",
+  "ability.biomancy.fleshkin_affinity": "血肉亲和",
+  "ability.biomancy.fleshkin_affinity.desc": "+25%酸液抗性",
+  "ability.biomancy.bio_alchemical_epidermis": "生物炼金上皮",
+  "ability.biomancy.bio_alchemical_epidermis.desc": "+15%外部施加负面效果抗性\n +25%注射成功率\n -25%注射伤害                  ",
+  "ability.biomancy.imposing_aura": "庄严气场",
+  "ability.biomancy.imposing_aura.desc": "所有附近的生物受到虚弱I效果",
+  "ability.biomancy.primordial_sight": "原初视觉",
+  "ability.biomancy.primordial_sight.desc": "窥探温床内在",
+  "tooltip.biomancy.fire_rate": "射速",
+  "tooltip.biomancy.accuracy": "精准度",
+  "tooltip.biomancy.ammo": "弹药",
+  "tooltip.biomancy.reload_time": "装弹时间",
+  "tooltip.biomancy.projectile_damage": "伤害",
+  "tooltip.biomancy.projectile_knock_back": "击退",
+  "tooltip.biomancy.living_tool_state_is": "这把器械正处于%1$s",
+  "state.biomancy.living_tool.broken": "损坏状态",
+  "state.biomancy.living_tool.dormant": "休眠状态",
+  "state.biomancy.living_tool.awakened": "觉醒状态",
+  "msg.biomancy.not_sleepy": "困意一扫而空",
+  "msg.biomancy.not_enough_ammo": "弹药不足",
+  "msg.biomancy.not_enough_nutrients": "营养不足",
+  "msg.biomancy.not_enough_blood_charge": "血能不足",
+  "msg.biomancy.mob_too_old": "这个生物太老了",
+  "msg.biomancy.already_full": "已经满了",
+  "msg.biomancy.failed_to_store_mob": "储存生物失败",
+  "msg.biomancy.failed_to_spawn_mob": "释放存储的生物失败",
+  "msg.biomancy.item_interaction_not_allowed": "物品交互不允许",
+  "keyboard.biomancy.ctrl": "Ctrl",
+  "keyboard.biomancy.alt": "Alt",
+  "keyboard.biomancy.shift": "Shift",
+  "keyboard.biomancy.right_mouse": "鼠标右击",
+  "bio_forge_tab.biomancy.search": "全部",
+  "bio_forge_tab.biomancy.blocks": "建筑方块",
+  "bio_forge_tab.biomancy.machines": "机器",
+  "bio_forge_tab.biomancy.tools": "工具",
+  "bio_forge_tab.biomancy.components": "材料",
+  "bio_forge_tab.biomancy.misc": "杂项",
+  "jei.biomancy.recipe.bio_lab": "生物炼金釜",
+  "jei.biomancy.recipe.decomposer": "分解体",
+  "jei.biomancy.recipe.digester": "消化体",
+  "jei.biomancy.recipe.bio_forge": "生物锻造台",
+  "sounds.biomancy.ui.button.click": "血肉按钮：咔哒",
+  "sounds.biomancy.ui.menu.open": "血肉界面：打开",
+  "sounds.biomancy.ui.radial_menu.open": "径向菜单：打开",
+  "sounds.biomancy.item.injector.inject": "注射血清",
+  "sounds.biomancy.item.injector.fail": "注射失败",
+  "sounds.biomancy.item.bone_marrow.drink": "嘬吸骨髓",
+  "sounds.biomancy.item.impaler.shoot": "射出穿刺物",
+  "sounds.biomancy.item.impaler.hit": "穿刺炮：击中",
+  "sounds.biomancy.item.impaler.impact": "生物：碰撞",
```

Comment:
The sound 'impaler.impact' should play when I hit a mob and it impacts to block, right? 
So should it be called 'mob impact' instead?

Reply:
Actually i used similar naming scheme to minecraft. And the impact sound is actually for block hits and the hit sound is for mobs.